windows/devices/uwb/include/windows/devices/uwb/UwbConnector.hxx

```diff
@@ -143,7 +143,7 @@ public:
     virtual std::future<std::tuple<::uwb::protocol::fira::UwbStatus, std::optional<uint32_t>>>
     SessionGetRangingCount(uint32_t sessionId) override;
 
-    virtual std::future<::uwb::protocol::fira::UwbSessionUpdateMulicastListStatus>

```

Comment:
If we are changing this to `UwbStatus`, then is `UwbSessionUpdateMulicastListStatus` still needed anywhere? If not, we should remove it completely. If it is needed somewhere, we should probably fix the typo (`Mulicast`) at some point :)

Reply:
+1, let's remove it. Feel free to do it in a separate PR if that's easier.

Reply:
UwbSessionUpdateMulticastListStatus is still used in DispatchCallbacks, as it is relevant when we handle the SESSION_UPDATE_CONTROLLER_MULTICAST_LIST_NTF

Reply:
You're completely right. I mistakenly thought `UwbSessionUpdateMulticastListStatus` was comprised of a `UwbStatus` and a `std::vector<stuff>`, but actually it has the session id in it, not `UwbStatus`, so we absolutely need to keep it.

Reply:
I created an issue to track the typo: https://github.com/microsoft/nearobject-framework/issues/249